src/hotspot/share/opto/vtransform.cpp

```diff
@@ -1421,8 +1421,9 @@ void VTransformReinterpretVectorNode::print_spec() const {
 
 void VTransformBoolVectorNode::print_spec() const {
   VTransformVectorNode::print_spec();
-  const BoolTest bt(_test._mask);
-  tty->print(" test=");
+  BoolTest::mask m = BoolTest::mask(_test._mask & ~BoolTest::unsigned_compare);
+  const BoolTest bt(m);
+  tty->print(" test=%s", m == _test._mask ? "" : "unsigned ");
   bt.dump_on(tty);
```

Comment:
I was wondering why we pass the raw mask around instead of keeping it encapsulated in a BoolTest object.  Elsewhere I saw code like this: 
`cond->get_con() & (BoolTest::unsigned_compare - 1)`
which seems to be making fragile assumptions about BoolTest internals.